vignettes/01_rphenoscape.Rmd

```diff
@@ -159,19 +151,19 @@ In the above step you retrieved a synthetic matrix for the Ictaluridae.  Now you
 First, we need to parse the list to determine which taxa in the matrix (‘m’) are descendants of *Ictalurus*. This returns a list of TRUE or FALSE assignments for each taxon in the matrix:
 
 ```{r}
-is_desc <- pk_is_descendant('Ictalurus', m$taxa)
+is_desc <- is_descendant('Ictalurus', m$taxa)
 ```
 We then subset the matrix m to the descendants of *Ictalurus*:
 
 ```{r}
 ictalurus_m <- m[is_desc, ]
 ```
-View the new matrix by clicking on the ‘ictalurus_m’ ```data.frame``` in the Global Environment.
+View the new matrix by clicking on the ‘ictalurus_m’ `data.frame` in the Global Environment.
 
-We can also subset the matrix by anatomy. In this case, let’s select characters for the pectoral fin and its parts. We will use `pk_is_descendant()` to first parse the characters (column names; note the first three taxon-related columns are excluded) to determine which are descendents (types or parts) of the term ‘pectoral fin’. Because this function only looks for descendent terms, we also need to specify that any characters pertaining to 'pectoral fin' itself are included. 
+We can also subset the matrix by anatomy. In this case, let’s select characters for the pectoral fin and its parts. We will use `is_descendant()` to first parse the characters (column names; note the first three taxon-related columns are excluded) to determine which are descendents (types or parts) of the term ‘pectoral fin’. Because this function only looks for descendent terms, we also need to specify that any characters pertaining to 'pectoral fin' itself are included. 
```

Comment:
```suggestion
We can also subset the matrix by anatomy. In this case, let’s select characters for the pectoral fin and its parts. We will use [`is_descendant()`](https://rphenoscape.phenoscape.org/reference/is_relation.html) to first parse the characters (column names; note the first three taxon-related columns are excluded) to determine which are descendents (types or parts) of the term ‘pectoral fin’. Because this function only looks for descendent terms, we also need to specify that any characters pertaining to 'pectoral fin' itself are included. 
```